### PR TITLE
Update of ReadMe and Vignettes

### PR DESCRIPTION
**Pull request by LorenzoMerotto**
Update of the Readme/vignettes as described in #93 
I have not created the "mouse" specific vignette but I guess it can be added in a second time

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,22 @@
...
+Deconvolution of human data:
+
 ```R
 immunedeconv::deconvolute(gene_expression_matrix, "quantiseq")
 ```
 
+Deconvolution of mouse data:
+
+```R
+immunedeconv::deconvolute_mouse(gene_expression_matrix, "quantiseq")
```
Comment (grst) [inline README.md:22]: is quantiseq a good example here?

File: README.md [hunk 2]
```diff
@@ -9,12 +9,22 @@
...
+```
+
 where `gene_expression_matrix` is a matrix with genes in rows and samples in columns. The rownames must be
-[HGNC](https://www.genenames.org/) symbols and the colnames must be sample names. The method can be one of
+[HGNC](https://www.genenames.org/) symbols and the colnames must be sample names. For human data, the method can be one of
```
Comment (grst) [inline README.md:26]: for mouse, it's [MGI](http://www.informatics.jax.org/mgihome/nomen/) gene symbols, I guess?

File: README.md [hunk 3]
```diff
@@ -23,6 +33,24 @@ cibersort_abs
...
+
+The methods available for the deconvolution of mouse data are 
+
+```
+mmcp_counter
+seqimmucc
+dcq
+base
+```
+In addition, human-based methods can be used to deconvolute mouse data through the conversion of gene names
```
Comment (grst) [inline README.md:48]:
```suggestion
In addition, human-based methods can be used to deconvolute mouse data through the conversion to orthologous gene names
```

File: README.md [hunk 4]
```diff
@@ -37,24 +65,29 @@ Note that, while *immunedeconv* itself is free ([BSD](https://github.com/icbi-la
...
 ### Comparison of the methods
-For a benchmark comparison of the methods, please see our [publication](https://doi.org/10.1101/463828).
+For a benchmark comparison of the human-based methods, please see our [publication](https://doi.org/10.1101/463828).
 If you would like to benchmark additional methods, please see our [benchmark
 pipeline](https://github.com/icbi-lab/immune_deconvolution_benchmark).
 
 
 ## Installation
-System requirements: R >= 3.5. Only linux is officially supported, but Mac/Windows should work, too.
+System requirements: R >= 4.0. Only linux is officially supported, but Mac/Windows should work, too.
```
Comment (grst) [inline README.md:90]:
```suggestion
System requirements: R >= 4.1. Only linux is officially supported, but Mac/Windows should work, too.
```

Good catch, though it's even 4.1 by now!

**Review by grst — COMMENTED** [review]
Looks great, just a few minor things!

File: README.md (hunk 1 above)
Comment (LorenzoMerotto) [reply README.md:22]: No it's a mistake
I got confused with the section where I make the conversion of gene names

**Pull request merged**